src/documents/Sonification.md

```diff
@@ -0,0 +1,63 @@
+
+*Data sonification* is an analogous audio technique to *data visualization* where structured data is converted into sound to try to:
+* communicate aspects of often-complex information clearly and efficiently
+* support specific analysis of aspects of the data
+* show patterns and relationships in the data
+
+## The Case for Sonification in Open Educational Resources
+
+While sonification is traditionally a theoretical and experimental area, it has rich potential as a means of supporting ["the creation of variations and enhancements"](/Techniques.html#AllowTheCreationOfVariationsAndEnhancements) in OER.
+
+Sonification's most obvious audience would seem to be the learner with visual impairments, but it can also be appreciated by those with cognitive impairments, learning disabilities, or a preference for audio presentation. It can help supplement data visualizations or open new areas for discussion, research and learning.
+
+### Floe Work
+
+The Floe team is experimenting with sonification and building authoring tools to support it as part of its work in exploring multimodal content creation and presentation in OERs.
+
+The <a href="http://build.fluidproject.org/chartAuthoring/demos/" target="_blank" class="link-external">Floe chart authoring tool</a> is a multimodal authoring tool that produces three forms of content from user-entered data data:
+
+* A pie chart (data visualization)
+* A data table that also functions as the pie chart's legend
+* A sonified form of the data (data sonification)
+
+### Sonification, Visualization and Underlying Data
+
+Placing sonification alongside the more familiar paradigm of data visualization explicitly surfaces that charts and graphs are a derived representation of a data set, and that one way OERs can support diverse learners and meet the [Accessibility Principles](/AccessiblityPrinciples.html) of perceivability and understandability is to make underlying data sets available wherever possible when presenting information in charts, graphs or other derived forms.
+
+When underlying data is available in OERs, this allows the possibility for others to transform the OER in ways the content creator may not have envisioned in order to best serve individual learner needs.
+
+## Exploring Sonification and Emerging Best Practices
+
+Sonification research to date has been highly experimental, and we still aren't certain how best to use sonification as part of building inclusive OERs. However, some general guidelines that have emerged:
+
+* As with other forms of presentation, support the possibility of adaptation to individual learner needs by building with open frameworks and approaches, and preserving access to source data.
+* Explore various facets of audio perception when building sonifications, such as pitch, loudness and duration.
```

Comment:
Is it worth mentioning spatial arrangement too? (i.e. left-right, surround / virtual sound environments).